include/pybind11/detail/class.h

```diff
@@ -87,6 +90,13 @@ inline PyTypeObject *make_static_property_type() {
         pybind11_fail("make_static_property_type(): failure in PyType_Ready()!");
     }
 
+#    if PY_VERSION_HEX >= 0x030C0000
```

Comment:
```suggestion
#if PY_VERSION_HEX >= 0x030C0000
```

Reply:
I just tried this change locally, but pre-commit clang-format changes it straight back the way it was (with the spaces that you want to delete here).

I'm confused ... did you assume clang-format accepts it either way? Or is there something else I'm missing?

Reply:
Okay, though it didn't format this since I saw code below without it, okay if it's being autoformatted!